src/wazuh_modules/vulnerability_detector/cpe_helper.json

```
@@ -561,9 +616,11 @@
                     "^Python"
                 ],
                 "product": [
-                    "^Python [0-9]\\.[0-9]\\.*[0-9]* \\("
+                    "^Python [0-9]\\.[0-9]+\\.*[0-9]* \\("
                 ],
-                "version": [],
+                "version": [
+			"^Python ([0-9]\\.[0-9]+\\.[0-9])"
```

Take care that the indentation is similar to the other cases:
```suggestion
                    "^Python ([0-9]\\.[0-9]+\\.[0-9])"
```